src/superphot_plus/ztf_transient_fit.py

```diff
@@ -179,9 +182,13 @@ def create_logL(cube):
         float
             Log-likelihood value.
         """
-        f_model = flux_model(cube, tdata, bdata)
+        f_model = flux_model(cube, tdata, bdata, unique_bands, ref_band)
         extra_sigma_arr = np.ones(len(tdata)) * cube[6] * max_flux
-        extra_sigma_arr[bdata == "g"] *= cube[13]
+        
+        for e, ub in enumerate(unique_bands):
```

Comment:
Same comment as earlier that `e` and `ub` are confusing variable names. I would particularly recommend changing `e` because it can be confused with scientific notation.